sky/packages/sky/lib/src/animation/curves.dart

```diff
@@ -179,3 +179,6 @@ const ElasticOutCurve elasticOut = const ElasticOutCurve();
 
 /// An oscillating curve that grows and then shrinks in magnitude while overshootings its bounds
 const ElasticInOutCurve elasticInOut = const ElasticInOutCurve();
+
+/// A curve that starts quickly and eases into its final position. Over the course of the animation, the object spends more time near its final destination. As a result, the user isn’t left waiting for the animation to finish, and the negative effects of motion are minimized.
```

Comment:
Can you wrap this comment art 80 cols?  We're not super strict about that, but this one goes on for a while.  :)